d3ploy/DemandFac.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import scipy as sp
+import random
+
+from cyclus.agents import Facility
+from cyclus import lib
+import cyclus.typesystem as ts
+
+
+class DemandFac(Facility):
+    """
+    This institution deploys facilities based on demand curves using 
+    Non Optimizing (NO) methods. 
+    """
+
+    demand_rate_min=ts.Double(
```

Comment:
Variable assignment has a single space on both sides of the equal sign.

Reply:
Yesterday before I put this in you told me to remove the space on either side of the equal signs. We specifically looked at the example in the python developer guide.

Reply:
Yes, PEP8 says to not have spaces between equal signs inside of function calls, method calls, and class instantiations.  However, assignment operations (`x = 10`) should have a single space on each side.  This is how you tell if something is in a function call or a variable. Hopefully this clarifies it!

Reply:
Also there are a number of tools like pep8 and flake8 and pylint and pyflakes that will tell you where the PEP8 violations are in a file / fix them for you.